90zfsbootmenu/zfsbootmenu-lib.sh

```diff
@@ -1413,14 +1413,18 @@ import_pool() {
 
   zdebug "zpool import arguments: ${import_args[*]} ${pool}"
 
-  zpool import "${import_args[@]}" "${pool}" >/dev/null 2>&1
+  import_output="$(
+    zpool import "${import_args[@]}" "${pool}" 2>&1
+    exit $?
```

Comment:
Doesn't the return value of `zpool import` fall through to be the return value of the `import_output=` assignment without needing `exit $?`?

Reply:
You're right, the last return code in a subshell is the exit code of the subshell.